trade-finance-manager-ui/server/controllers/utilisation-reports/add-payment/index.ts

```diff
@@ -49,7 +56,15 @@ const extractFormValuesFromRequestBody = (requestBody: AddPaymentRequestBody): A
 
 const renderAddPaymentPage = (res: Response, context: AddPaymentViewModel) => res.render('utilisation-reports/add-payment.njk', context);
 
-const mapToSelectedReportedFeesDetailsViewModel = (selectedFeeRecordData: SelectedFeeRecordsDetails): SelectedReportedFeesDetailsViewModel => {
+const mapToRecordedPaymentDetailsViewModel = (payment: SelectedFeeRecordsPaymentDetailsResponse): RecordedPaymentDetailsViewModel => {
+  return {
+    reference: payment.reference,
+    value: getFormattedCurrencyAndAmount(payment.value),
```

Comment:
Could we call this `formattedCurrencyAndAmount` to match other places where we use this?